src/components/Chips/Chips.tsx

```diff
@@ -17,14 +17,10 @@ import useSetFocus from "../../hooks/useSetFocus";
 import useClickableProps from "../../hooks/useClickableProps/useClickableProps";
 import useIsOverflowing from "../../hooks/useIsOverflowing/useIsOverflowing";
 import useChipOverflowTooltip from "./hooks/useChipOverflowTooltip";
-import { BEMClass } from "../../helpers/bem-helper";
 import { ElementContent } from "../../types/ElementContent";
-import "../Clickable/Clickable.scss";
+import clickableStyles from "../Clickable/Clickable.module.scss";
```

Comment:
importing the same scss file for different component is currently not our coding best practice. At the moment we are using mixins for reuse styles. Let's change it to use 3 global mixins: one for clickable, one for clickable disabled and one for disableTextSelection

Reply:
done